src/frontend/src/views/Data/components/SamplesView/components/SamplesTable/components/LineageTooltip/components/GeneralViralLineageTooltip/index.tsx

```diff
@@ -0,0 +1,73 @@
+import { Tooltip } from "czifui";
+import { LineageTooltipProps } from "../../lineageTooltipConfig";
+import { Label, Text, Wrapper } from "./style";
+
+const KEY_TO_LABELS = {
+  lineage: "Lineage",
+  qc_status: "QC Status",
+  scorpio_call: "Scorpio Call",
+  scorpio_support: "Scorpio Support",
+  lineage_software_version: "Version",
+  lineage_type: "Lineage Type",
+  lineage_probability: "Lineage Probability",
+  last_updated: "Last Updated",
+  reference_dataset_name: "Nextclade Dataset",
+  reference_sequence_accession: "Reference Sequence Accession",
+  reference_dataset_tag: "Reference Dataset Tag",
+};
+
+const DISPLAY_ORDER: Array<keyof Lineage> = [
+  "lineage",
+  "qc_status",
+  "reference_dataset_name",
+  "lineage_software_version",
+  "lineage_type",
+  "lineage_probability",
+  "last_updated",
+  "scorpio_call",
+  "scorpio_support",
+  "reference_sequence_accession",
+  "reference_dataset_tag",
+];
```

Comment:
I'm using the same pattern from the existing tooltip of listing all of the fields and then hiding the ones we don't need.  I'm not sure if this is confusing and/or why it was done this way in the first place.  Thought?